### DISCUSSION TITLE
Introduce yourself to the OceanGliders Community

### DISCUSSION THREAD

**Discussion by soerenthomsen** [OceanGlidersCommunity/OceanGliders#1] [Announcements]
## Dear OceanGliders Community
We’re using Discussions as a place to connect with other members of our community. 

In this thread we introduce ourselves and let the others know what we do.

Please only stick to introducing yourself and open an new discussion or issues for other topic to keep this thread clean! 

Thank you! 

PS: Also please read our [Code of Conduct](https://github.com/OceanGlidersCommunity/OceanGliders/blob/main/CODE_OF_CONDUCT.md)

@jbusecke @ptestor @vturpin @isgiddy and others please consider to introduce yourselves here

**Comment by tcarval** [comment]
I am a data manager in Ifremer (France) , involved in Argo and EU glider data activities

**Comment by soerenthomsen** [comment]
Hi everyone,

I am Soeren Thomsen, Physical Oceanographer. Right now I work 50 % as OceanGliders.org Best Practice coordinator.

I have been working with Slocum gliders since around 2011.
For my PhD (2012-2016) and PostDoc at GEOMAR (2017-2018) I designed and carried out two glider fleet experiments off Peru (https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2015JC010878, https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2016GL070548) and one Mauritania (https://bg.copernicus.org/articles/16/979/2019/).
Oxygen measurements where a key part of these experiments. Luckily I got support from @hahn-johannes and @gkrahmann for the oxygen calibration.

I am a big fan of open science. So all related datasets are quality controlled open access available on Pangaea (https://doi.pangaea.de/10.1594/PANGAEA.860727, https://doi.pangaea.de/10.1594/PANGAEA.860480) thanks to G. Krahmann and the data management team at GEOMAR 🙏. Please get in contact if you are interested in glider datasets in upwelling systems (i.e. off Peru, Mauritania, Namibia or Benguela). I can navigate you through the different available deployments available on Pangaea.

I switched to Python in 2018 mainly to analyse ROMS/CROCO model output using xarray, dask etc! So far I have little experience in collective coding but motivated to learn on the fly!

Looking forward to get to know the others of the OceanGlidersCommunity

Best
Soeren

PS: I nominate @callumrollo and @angelitora to introduce themselves. And please nominate 2 more so we get this started :)

**Comment by patricialg** [comment]
Hi all,
My name is Patricia and I work as Sensor Science Delivery Lead at NOC in Southampton. I'm currently working with nutrient sensors in the lab and as delivery lead for some projects like TechOceanS, AutoNuts, etc.. In these projects we develop sensors that are integrated in differnt platforms. In TechOceanS project, I'm also involved in the work related with the development of Best Practices for sensors.
I'm a marine scientist. I have extensive laboratory experience including some of the latest marine chemistry and trace analysis techniques.
From 2014-2019 I was responsible for collection and analysis of samples from the European Station for Time series in the Ocean Canary Islands (ESTOC) and the Test Site of PLOCAN in addition to the calibration and validation of sensors integrated in moorings and gliders. I used to work closely with glider pilots (I was the lab rat in a white/mostly white lab coat).
I'm contributing (or trying to contribute) to the Glider best practices document that Soeren is coordinating. I'm helping with the Oxygen sensors section. My main interest is to help, when possible, to calibrate/validate sensors so that they will produce the best possible quality data and find the best protocol for the gold standard method to achieve this. Gliders are the future of Ocean Observation but without good sensors, they are only fancy toys, so I really believe that this initiative is very important.
I'm happy to discuss, learn from others and look together for the best approaches :)
Cheers,
Patry

**Comment by callumrollo** [comment]
Hi all,

I'm the data manager/data scientist for the VOTO foundation in Sweden. We run continuous glider observatories in the Baltic; you can see our live data [here](https://observations.voiceoftheocean.org/). We started our observatories in late 2020, so we have a great opportunity to start our processing pipeline from scratch adopting best practices.

I recently completed a PhD with the UEA glider group. I am a dedicated Python programmer and linux nerd. I am a contributor to several Python based glider tools including:

- [pyglider](https://github.com/c-proof/pyglider)
- [GliderTools](https://github.com/GliderToolsCommunity/GliderTools)
- [gliderpy](https://github.com/ioos/gliderpy)

I want to get involved with improving the OceanGliders best practices documents and OG1.0 format. I look forward to learning from other data managers and their processing systems. I have previoulsy taught courses in Python and git within my university and with [The Carpentires](https://carpentries.org/). I'm always happy to help people navigate github and get involved in open source.

I nominate @elyfant and @justinbuck to introduce themselves.

**Comment by abussani** [comment]
Hi to all
I'm Antonio Bussani, I'm working as a pilot in OGS.
Also I work on the data flow of the gliders we have, and arranging the database with all the deployment. 
My work concern also about drifter and float, our work could be seen [here](http://argo.ogs.it)
Usually we publish our mission on ego platform and we create the DOI for each missions...
Feel free to contat us if you need something!

**Comment by hassanimaryam** [comment]
Hi everyone
I'm Maryam, Physical Oceanography student at the University of Tehran

**Reply by e-abdi** [reply]
Hi Maryam,
I am sorry you could not make it to glider school at PLOCAN last week! They told me your visa didn't come... I was representing Seaglider there. If you have any questions, or need assistance regarding gliders, I'll be happy to help. This is my email: e.abdi@cyprus-subsea.com
Ehsan

**Reply by hassanimaryam** [reply]
Thank you very much Ehsan for your message. Sorry, for the delayed answer. We have some internet issues here. I have sent you an email.

**Comment by catsch** [comment]
Hi all, 

I am Catherine Schmechtig, I work on [BGC-Argo data management](https://biogeochemical-argo.org/) (Metadata, Real time, Delayed mode) since 2011 (but I have a PhD in atmospheric sciences so still a lot to learn !!!). Since 2011, I ve been working a lot to push BGC data into the Argo Data system and I will be happy to share.

**Comment by sophieclayton** [comment]
Hi all,
I'm an assistant professor of Oceanography at Old Dominion University in the US. I work with biogeochemical data collected on a range of platforms including moorings, gliders and Argo floats. 
I am also co-lead of a [working group developing best practices for using BGC data generated by the OOI arrays](https://www.us-ocb.org/ooi-dataset-community/).
I am a big advocate of open science and data democratisation, so I think it's very important to get these kinds of best practices out to the community so that more people can use this type of data!

@flane00 introduce yourself here!

**Comment by angelitora** [comment]
Hi everyone, 
My name is Angel and I am an Oceanographer and I have been one year working as an Associate Professor at the [University of Iceland](https://english.hi.is/). In the past I worked at the University of Mexico ([UNAM](https://www.unam.mx/)) where I had the opportunity to join the one and only Mexican glider team in the Gulf of Mexico ([GMOG](https://gliders.cicese.mx/)). We had lots of fun looking at Loop Current Eddies and enjoying building up the glider team, we are still working together. Currently, I am looking forward to start some glider section in the South of Iceland and we are starting to build a team at the University. Iceland is small and critical mass is an issue; thus, all collaborations are very valuable :)
I am very interested in learning and implementing best practices for data sharing and looking forward to an open science data future. I also enjoy very much baking and biking! 
Good to meet y'all and I am afraid I was the last so none more on the list to nominate?

**Comment by justinbuck** [comment]
Hi All,

I am @justinbuck , a senior data manager in the National Oceanography Centre UK, British Oceanographic data Centre (BODC). In the context of gliders and the GOOS i have worked on a range of activities including OceanGliders, EGO, Argo, and OceanSITES. My roles in the context of gliders are broad I am currently the for systems development of BODC glider systems, working on the GROOM II data roadmap, contributing to the new IOC data policy, and co-investigator of the projects developing end to end data systems for autonomous platform data in NOC. As part of the the recent OceanOBS'19 work I contributed o white papers on best practices, autonomy, FAIR data and led the white paper on data innovation.

My motivation for being part of OceanGilders to to contribute to the development of and integrated component of the GOOS that is capable of developing at the same pace as the observational capabilities and is easy for the broaid range of users to interact with.

I am looking forward to the continuing collaboration with you all,
Justin

**Comment by davehebert** [comment]
I am Dave Hebert, a physical oceanographer at the Bedford Institute of Oceanography, Fisheries and Oceans Canada (DFO).  I work mainly on our Atlantic Zone Monitoring Program.  I oversee our coastal ocean SeaExplorer gliders.  I am also on the OceanGliders Steering Committee.

**Comment by ptestor** [comment]
I am Pierre Testor, a physical oceanographer at LOCEAN/CNRS, Paris, France. I have been the first in Europe to deploy gliders in 2004 and have participated in the scientific animation of the worldwide glider community since then, in particular through the EGO network (https://www.ego-network.org/). I have helped to launch OceanGliders as an associated program of the GOOS in 2016 and am presently co-chair of it. Scientifically, I am concerned with ocean variability with emphasis on observations of ocean circulation and mixing using various platforms, and gliders in particular. I am more specifically interested in 1) the regional oceanography of the Mediterranean Sea and the development of long-term ocean observatories in this hot spot of climate change and biodiversity (I am co-coordinating a northwestern component, Mediterranean Ocean Observing System for the Environment, MOOSE https://www.moose-network.fr/fr/ ), 2) the mesoscale and submesoscale phenomena and their role in the cross-slope exchanges, the winter water formation, and the physical-biological coupling.

**Comment by jenseva** [comment]
I’m Jenn Sevadjian and I’ll be working on data management for Spray gliders. I look forward to contributing to this group!

**Reply by soerenthomsen** [reply]
Thanks for joining @jenseva

**Comment by sfregosi** [comment]
Hello, I am Dr. Selene Fregosi and I am a contractor for the U.S. NOAA Pacific Islands Fisheries Science Center. I work with passive acoustic Seagliders to listen for and count whales. Looking forward to learning from this community!

**Reply by soerenthomsen** [reply]
Thanks for joining @sfregosi !

**Comment by jstorylong** [comment]
Hi OG community! I'm Jacki Long, a data scientist at the Monterey Bay Aquarium Research Institute where we're deploying gliders with novel payloads and instruments off the California coast! Thanks to the moderators for bringing us all together :)

**Reply by soerenthomsen** [reply]
Welcome @jstorylong thanks for joining!

**Comment by Zengirl2** [comment]
Hey there, I'm Leslie Birch and I'm currently at Rutgers for physical oceanography. My focus is AUVs, so I'm training on a REMUS, Slocum gliders, and hopefully a Seaber soon :). I'm interested in backseat capability and also an advocate of open-source.

**Reply by soerenthomsen** [reply]
Hi @Zengirl2 

great to have you on board and thanks for introducing yourself. Feel free to join the discussions and contribute to the repos of your interest. Happy to read that you are advocating for open-source :)

**Comment by fmargirier** [comment]
Hi All !
I'm Félix Margirier, and started working with gliders during my PhD when I focused on the Northwestern MedSea and convective plumes at LOCEAN. I now work at ALSEAMAR (the SeaExplorer manufacturer), but am still very much doing research linked to different gliders.

Several of us have been exchanging through email about vertical velocities associated with gliders and we realised this would be the right place to share info, code and data as well as bringing more people together.

Looking forward to interact more with everyone !

**Reply by vturpin** [reply]
Hi Felix,

Welcome and thanks for the introduction. 
I am not sure there is yet a topic around vertical velocities. Maybe you could create a repository to share the thoughts with the OceanGliders community?
Would a Standard Operating Procedure could suite to this topic to guide the community on the practices to compute vertical velocities?
I would also suggest to invite people interested on board of this discussion.
I can assit you if needed (at least a minimum).

Looking forward to see vertical velocity topic building up ;)

**Reply by fmargirier** [reply]
Hi Victor !
Thank for the welcome. I guess an SOP would be part of the discussion so yes that is what i'll set up, thanks for the suggestion.
Our working group will definitely all join and of course anyone is very welcome too.
thanks again !

**Comment by jcermauwedu** [comment]
Hi All! My name is Rob Cermak.  I was a data manager for four years with AOOS as IOOS/RAs were largely spinning up.  I am primarily a computer programmer (python, et. al.) and system administrator with a Meteorology background turned to Oceanography and assist my wife with MOM6 ocean modeling projects.  The task before me, with a data provider hat, is adapting active acoustic and optical datasets to data pipelines so they comfortably flow into everyones workflows, portals and hopefully the NCEI archive.  For now the data originates from a University of Alaska College of Fisheries and Ocean Sciences Slocum Glider (unit_507 a.k.a. Gretel) and another Slocum in the near future.  Look forward to contributing what we can to move things forward and get things connected.

**Comment by kittykill-exe** [comment]
Hi all! I am Jia, nice to meet you! I am student studying in The Netherlands. I have done courses in data science, programming and wish to continue my studies to do research masters in data science. I am also learning about high performance computing and I wish to do a research project applied to ocean observation and decision making. My motivation for joining Ocean Gilders community to to contribute to the development of data practice and learn as much as I can. I also feel supportive of open data and I am grateful for groups like this that make collaboration possible. I also enjoy exploring new places and trying the best restaurants, so love to hear recommendations. I am interested in learning more from this community and am excited to collaborate with you.

**Reply by vturpin** [reply]
Hi Jia,
Welcome in the OceanGliders community github.
Take time to discover the different repositories and do not hesitate to ask questions if you have any.
And good luck with your masters degree.

**Comment by naomikrauzig** [comment]
Hi everyone,
I’m Naomi Krauzig, observational physical oceanographer at GEOMAR and Chair of the OceanGliders Polar Task Team. I just moved to the Ocean Circulation and Climate Dynamics division at GEOMAR where I will have the chance to work with a large Slocum fleet (and hopefully lots with the MicroRider). My first experience was with SeaExplorers working on shelf–basin exchange, ocean-ice interactions, and water-mass variability using gliders, under-ice Argo floats, moorings and ship-based observations.

I’m a strong advocate for FAIR and CARE data principles and open science from data acquisition to publication, and I’m keen to contribute to SOPs, best-practice documents, and the OG1.0 format. 

As a data enthusiast, I’m especially looking forward to learning from data managers and exchanging ideas within the OceanGliders community.

Cheers,
Naomi

**Reply by jenseva** [reply]
Welcome, Naomi! 

Thank you for reaching out to participate in our activities. I will add you to our list of contacts for the data management team and we welcome you to join future discussions. We meet ad hoc as needed and welcome community participation both in our meetings and through GitHub channels.  We will be scheduling a data team meeting soon, hope to see you then. 
Best, Jenn

**Comment by e-abdi** [comment]
Hello community,

I'm Ehsan Abdi, a glider engineer at Akvaplan-niva, a research institute in Norway. I have been working with gliders since 2015. After the recent UG2 meeting I got motivated to contribute to this effort especially to the operational side of things,  since I see that there are a lot of new people entering glider world and there it can be quite overwhelming and confusing to try to figure everything out from zero. I have started a few personal repositories to share my experience especially when it comes to custom sensor integration but here I would like to initiate something like "operations" user group which can focus on good practices for mission planning, ballasting, piloting and basic glider maintenance.

**Reply by callumrollo** [reply]
Hi @e-abdi great to have you here! I think some operational knowledge would be really useful to structure and share. Are you in the UG2 slack already? That's a great place for techinical questions, especially with slocum gliders. I think a more structured wiki-like resource for ops knowledge would be a very useful project

**Reply by e-abdi** [reply]
Thanks @callumrollo! Yes, I’ve been following and contributing on the UG2 Slack for the last few years, and there’s so much valuable knowledge shared there. I think as you said, it would be really useful to organize and structure that information better so it’s easier for everyone to access and build on. That’s what I’m hoping to help with.

**Reply by elyfant** [reply]
Hey Ehsan, this sounds like a great step organising our collective operatinal knowledge into a more searchable format. From our chat on friday it sounds like you were talking about about 'Oceangliders' style SOPs for standard glider platform processes and then a wiki, for well, everything. Is OceanGliders the appropriate place for this @callumrollo ?

We have lots of documentation to contribute for seagliders and slocums

**Reply by callumrollo** [reply]
I think OceanGliders is definitely a good place to collect this knowledge! We just need to decide what format/contribution method works best. Feel free to ping me if you need a repo set up/a GitHub team made/extra permissions etc.

I made a very modest start on the data processing side of things with this collaborative list of glider tools https://github.com/OceanGlidersCommunity/glider-tools-list

I think a similar (though, by its nature larger and more complex) effort for practical glider operations would be fantastic

**Reply by e-abdi** [reply]
that sounds good! we will create a draft of what we have in mind with Fiona and forward it to you. Once agreed, we can use that as a starting point and build on it together with the rest of the community.
